recipes/watchdog/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "watchdog" %}
+{% set version = "0.8.3" %}
+{% set hash_type = "md5" %}
+{% set hash = "bb16926bccc98eae2a04535e4512ddf1" %}
```

Comment:
We prefer `sha256` if that is an option. This can normally be gotten from [pypi.io]( https://pypi.org/ ).